### PR TITLE
Feat/landing

### PR DESCRIPTION
**Pull request by Hakdoff**
Overview
- Added landing page

Note
- SCSS file cannot add line break (automatically removed after saving)

Snippet
image

### CODE REVIEW

File: src/resources/scss/app.scss [hunk 1]
```diff
@@ -3,3 +3,4 @@
 @use './dashboard/dashboard';
 @use './post/post';
 @use './profile/profile';
+@use './landing/landing';
```
Comment (hipe01mark) [inline src/resources/scss/app.scss]: BR

File: src/resources/scss/landing/landing.scss [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    }
+
+    &__footer-logo {
+        @apply font-lora items-center font-lora font-semibold text-3xl flex justify-center gap-3 text-[#333333];
+    }
+
+    &__footer-all {
+        @apply justify-center items-center font-medium text-base flex text-[#333333] gap-3;
+    }
+}
```
Comment (hipe01mark) [inline src/resources/scss/landing/landing.scss]: BR

File: src/resources/views/pages/landing/landing-page.blade.php [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+            <div class="l-landing__container">
+                <nav class="l-landing__nav">
+                    <div class="l-landing__logo">
+                        @include('/svg/logo-nav')  
+                        <a href="/">Microblog</a>
+                    </div>
+                    <div class="grid-cols-3"></div>
+                    <div class="l-landing__login">
+                        <div class="font-normal">
+                            <a href="/login">Login</a>
```
Comment (hipe01mark) [inline src/resources/views/pages/landing/landing-page.blade.php]: use route()

File: src/resources/views/pages/landing/landing-page.blade.php [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+                        </div>
+                        <div class="font-normal">
+                            <a href="/register-page">Register</a>
```
Comment (hipe01mark) [inline src/resources/views/pages/landing/landing-page.blade.php]: use route()

File: src/resources/views/pages/landing/landing-page.blade.php [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+                </div>
+                <div class="l-landing__developers-position-container">
+                    <p class="l-landing__developers-position">Software Engineer Intern - Full Stack</p>
+                </div>
+            </div>
+        </div>
+        <nav class="l-landing__footer">
+            <div class="l-landing__footer-logo">    
+                 @include('/svg/logo-nav')  
+                <a href="/login">Microblog</a>
```
Comment (hipe01mark) [inline src/resources/views/pages/landing/landing-page.blade.php]: use route()

File: src/resources/views/pages/landing/landing-page.blade.php [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+@section('title', 'MicroBlog')
+
+@section('content')
+    <body>
+        <div style="background-image: url('/img/bg.jpg')" class="l-landing__bg-img">
+            <div class="l-landing__container">
+                <nav class="l-landing__nav">
+                    <div class="l-landing__logo">
+                        @include('/svg/logo-nav')  
+                        <a href="/">Microblog</a>
```
Comment (hipe01mark) [inline src/resources/views/pages/landing/landing-page.blade.php]: use route()

**Comment by hipe01mark** [comment]
If the saved images are on purpose or for the sake of the UI design. Kindly optimize it first then convert it to webp.

File: src/resources/scss/landing/landing.scss [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+    &__caption {
+        @apply flex pt-32 h-44 justify-center items-center mx-auto space-y-4;
+    }
+
+    &__subtitle {
+        @apply flex justify-center md:mx-auto items-center space-y-4 pt-7;
+    }
+
+    &__libre {
+        @apply flex text-center md:text-xl text-lg font-librefranklin text-[#333333];
```
Comment (mikco-hipe) [inline src/resources/scss/landing/landing.scss]: #333333 is recurring, maybe it is better to assign variable for this

File: src/resources/views/pages/landing/landing-page.blade.php [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+                <div class="l-landing__features-card-subtitle-container">
+                    <p class="l-landing__features-card-subtitle">Unfollow to keep your feed relevant.</p>
+                </div>
+            </div>
+        </div>
+        <div class="l-landing__joinus">
+            <div class="container mx-auto">
+                <div class="l-landing__joinus-container">
+                    <p class="l-landing__joinus-title">Be part of a supportive and engaging community. Whether you are here to share or to discover, our platform welcomes you.</p></p>
+                    <div class="l-landing__joinus-img-container">
```
Comment (mikco-hipe) [inline src/resources/views/pages/landing/landing-page.blade.php]: extra paragraph closing tag

**Review by mikco-hipe — COMMENTED** [review]
Please check!

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**